molgenis-data-security/src/main/java/org/molgenis/data/security/EntityTypeIdentity.java

```diff
@@ -0,0 +1,19 @@
+package org.molgenis.data.security;
+
+import org.molgenis.data.meta.model.EntityType;
+import org.springframework.security.acls.domain.ObjectIdentityImpl;
+
+public class EntityTypeIdentity extends ObjectIdentityImpl
+{
+	public static final String TYPE = "entityType";
+
+	public EntityTypeIdentity(EntityType entityType)
+	{
+		this(entityType.getId());
```

Comment:
Have you looked at `ObjectIdentityRetrievalStrategy`?

Reply:
Nope, I propose to look into it after merging this PR.